src/api/AuthServer/AuthServer/HostingExtensions.cs

```diff
@@ -63,29 +76,14 @@ public static WebApplication ConfigureServices(this WebApplicationBuilder builde
                 options.ClientSecret = "copy client secret from Google here";
             });
 
+        builder.Services.AddAutoMapper(typeof(MappingProfile));
 
-        // this adds the necessary config for the simple admin/config pages
-        {
-            builder.Services.AddAuthorization(options =>
-                options.AddPolicy("admin",
-                    policy => policy.RequireClaim("sub", "1"))
-            );
-
-            builder.Services.Configure<RazorPagesOptions>(options =>
-                options.Conventions.AuthorizeFolder("/Admin", "admin"));
+        builder.Services.AddControllers();
 
-            builder.Services.AddTransient<ClientRepository>();
-            builder.Services.AddTransient<IdentityScopeRepository>();
-            builder.Services.AddTransient<ApiScopeRepository>();
-        }
+        builder.Services.AddScoped<AuthService>();
 
-        // if you want to use server-side sessions: https://blog.duendesoftware.com/posts/20220406_session_management/
-        // then enable it
-        //isBuilder.AddServerSideSessions();
-        //
-        // and put some authorization on the admin/management pages using the same policy created above
-        //builder.Services.Configure<RazorPagesOptions>(options =>
-        //    options.Conventions.AuthorizeFolder("/ServerSideSessions", "admin"));
+        builder.Services.AddFluentValidation();
+        builder.Services.AddValidatorsFromAssemblyContaining<IAssemblyMarker>();
```

Comment:
Either rename `IAssemblyMarker` so that it reflects that it's located at the identity server assembly or use full namespace here